src/main/java/juja/sqlcmd/DatabaseManager.java

```diff
@@ -2,20 +2,41 @@
 
 import java.sql.Connection;
 import java.sql.DriverManager;
+import java.sql.ResultSet;
 import java.sql.SQLException;
+import java.sql.Statement;
 
 public class DatabaseManager {
     private static final String DB_CONNECTION_URL = "jdbc:postgresql://127.0.0.1:5432/";
 
     private Connection connection;
 
-    boolean connect(String database, String user, String password) {
+    public boolean connect(String database, String user, String password) {
         try {
             Class.forName("org.postgresql.Driver");
-            connection = DriverManager.getConnection(DB_CONNECTION_URL + database, user, password);
+            connection = DriverManager.getConnection(DB_CONNECTION_URL + database+"?loggerLevel=OFF", user, password);
             return true;
         } catch (ClassNotFoundException | SQLException e) {
             return false;
         }
     }
+
+    public String[] getTableNames() throws SQLException {
+        String sqlQuery = "SELECT relname FROM pg_stat_user_tables";
+        try (Statement statement = connection
+                .createStatement(ResultSet.TYPE_SCROLL_INSENSITIVE, ResultSet.CONCUR_UPDATABLE);
+             ResultSet resultSet = statement.executeQuery(sqlQuery)) {
```

Comment:
Why after '{' you don't add  offset at the next line?

Reply:
With code formatting in that place everything is ok. Next line after '{' has offset from try() block which has ResultSet as a resource